### ISSUE TITLE
Cannot build v1.6.4 due to failed nats checkout

### ISSUE THREAD

**Issue by omgold** [influxdata/telegraf#4270]
### System info:

Linux, Go 1.10.3, Telegraf 1.6.4

### Steps to reproduce:

export GOPATH=\`pwd\`
git clone https://github.com/influxdata/telegraf src/github.com/influxdata/telegraf
cd src/github.com/influxdata/telegraf
git checkout 1.6.4
make

```
make deps
make[1]: Entering directory '/home/omangold/tmp/telegraf/mybuild/src/github.com/influxdata/telegraf'
go get -u github.com/golang/lint/golint
go get github.com/sparrc/gdm
gdm restore
======= Go Dependency Manager =======
= working dir:  /home/omangold/tmp/telegraf/mybuild/src/github.com/influxdata/telegraf
= checkout dir: /home/omangold/tmp/telegraf/mybuild/src
= GOPATH:       /home/omangold/tmp/telegraf/mybuild
=====================================
> Restoring /home/omangold/tmp/telegraf/mybuild/src/collectd.org to 2ce144541b8903101fb8f1483cc0497a68798122
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/aerospike/aerospike-client-go to 95e1ad7791bdbca44707fedbb29be42024900d9c
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/amir/raidman to c74861fe6a7bb8ede0a010ce4485bdbb4fc4c985
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/apache/thrift to 4aaa92ece8503a6da9bc6701604f69acf2b99d07
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/aws/aws-sdk-go to c861d27d0304a79f727e9a8a4e2ac1e74602fdc0
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/beorn7/perks to 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/bsm/sarama-cluster to abf039439f66c1ce78017f560b490612552f6472
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/cenkalti/backoff to b02f2bbce11d7ea6b97f282ef1771b0fe2f65ef3
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/couchbase/go-couchbase to bfe555a140d53dc1adf390f1a1d4b0fd4ceadb28
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/couchbase/gomemcached to 4a25d2f4e1dea9ea7dd76dfd943407abf9b07d29
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/couchbase/goutils to 5823a0cbaaa9008406021dc5daf80125ea30bba6
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/davecgh/go-spew to 346938d642f2ec3594ed81d874461961cd0faa76
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/dgrijalva/jwt-go to dbeaa9332f19a944acb5736b4456cfcc02140e29
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/docker/docker to f5ec1e2936dcbe7b5001c2b817188b095c700c27
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/docker/go-connections to 990a1a1a70b0da4c4cb70e117971a4f0babfbf1a
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/eapache/go-resiliency to b86b1ec0dd4209a588dc1285cdd471e73525c0b3
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/eapache/go-xerial-snappy to bb955e01b9346ac19dc29eb16586c90ded99a98c
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/eapache/queue to 44cc805cf13205b55f69e14bcb69867d1ae92f98
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/eclipse/paho.mqtt.golang to aff15770515e3c57fc6109da73d42b0d46f7f483
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/go-logfmt/logfmt to 390ab7935ee28ec6b286364bba9b4dd6410cb3d5
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/go-sql-driver/mysql to 2e00b5cd70399450106cec6431c2e2ce3cae5034
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/gobwas/glob to bea32b9cd2d6f55753d94a28e959b13f0244797a
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/go-ini/ini to 9144852efba7c4daf409943ee90767da62d55438
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/gogo/protobuf to 7b6c6391c4ff245962047fc1e2c6e08b1cdfa0e8
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/golang/protobuf to 8ee79997227bf9b34611aee7946ae64735e6fd93
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/golang/snappy to 7db9049039a047d955fe8c19b83c8ff5abd765c7
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/go-ole/go-ole to be49f7c07711fcb603cff39e1de7c67926dc0ba7
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/google/go-cmp to f94e52cad91c65a63acc1e75d4be223ea22e99bc
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/gorilla/mux to 392c28fe23e1c45ddba891b0320b3b5df220beea
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/go-redis/redis to 73b70592cdaa9e6abdfcfbf97b4a90d80728c836
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/hailocab/go-hostpool to e80d13ce29ede4452c43dea11e79b9bc8a15b478
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/hashicorp/consul to 63d2fc68239b996096a1c55a0d4b400ea4c2583f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/influxdata/tail to c43482518d410361b6c383d7aebce33d0471d7bc
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/influxdata/toml to 5d1d907f22ead1cd47adde17ceec5bda9cacaf8f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/influxdata/wlog to 7c63b0a71ef8300adc255344d275e10e5c3a71ec
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/fsnotify/fsnotify to c2828203cd70a50dcccfb2761f8b1f8ceef9a8e9
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/jackc/pgx to 63f58fd32edb5684b9e9f4cfaac847c6b42b3917
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/jmespath/go-jmespath to bd40a432e4c76585ef6b72d3fd96fb9b6dc7b68d
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/kardianos/osext to c2c54e542fb797ad986b31721e1baedf214ca413
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/kardianos/service to 6d3a0ee7d3425d9d835debc51a0ca1ffa28f4893
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/kballard/go-shellquote to d8ec1a69a250a17bb0e419c386eac1f3711dc142
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/matttproud/golang_protobuf_extensions to c12348ce28de40eed0136aa2b644d0ee0650e56c
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/Microsoft/go-winio to ce2922f643c8fd76b46cadc7f404a06282678b34
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/miekg/dns to 99f84ae56e75126dd77e5de4fae2ea034a468ca1
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/mitchellh/mapstructure to d0303fe809921458f417bcf828397a65db30a7e4
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/multiplay/go-ts3 to 07477f49b8dfa3ada231afc7b7b17617d42afe8e
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/naoina/go-stringutil to 6b638e95a32d0c1131db0e7fe83775cbea4a0d0b
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/nats-io/gnatsd to 393bbb7c031433e68707c8810fda0bfcfbe6ab9b
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/nats-io/go-nats to ea9585611a4ab58a205b9b125ebd74c389a6b898
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/nats-io/nats to ea9585611a4ab58a205b9b125ebd74c389a6b898
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/nats-io/nuid to 289cccf02c178dc782430d534e3c1f5b72af807f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/nsqio/go-nsq to eee57a3ac4174c55924125bb15eeeda8cffb6e6f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/opencontainers/runc to 89ab7f2ccc1e45ddf6485eaa802c35dcf321dfc8
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/opentracing-contrib/go-observer to a52f2342449246d5bcc273e65cbdcfa5f7d6c63c
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/opentracing/opentracing-go to 06f47b42c792fef2796e9681353e1d908c417827
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/openzipkin/zipkin-go-opentracing to 1cafbdfde94fbf2b373534764e0863aa3bd0bf7b
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/pierrec/lz4 to 5c9560bfa9ace2bf86080bf40d46b34ae44604df
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/pierrec/xxHash to 5a004441f897722c627870a981d02b29924215fa
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/pkg/errors to 645ef00459ed84a119197bfb8d8205042c6df63d
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/pmezard/go-difflib/difflib to 792786c7400a136282c1664665ae0a8db921c6c2
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/prometheus/client_golang to c317fb74746eac4fc65fe3909195f4cf67c5562a
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/prometheus/client_model to fa8ad6fec33561be4280a8f0514318c79d7f6cb6
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/prometheus/common to dd2f054febf4a6c00f2343686efb775948a8bff4
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/prometheus/procfs to 1878d9fbb537119d24b21ca07effd591627cd160
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/rcrowley/go-metrics to 1f30fe9094a513ce4c700b9a54458bbb0c96996c
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/samuel/go-zookeeper to 1d7be4effb13d2d908342d349d71a284a7542693
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/satori/go.uuid to 5bf94b69c6b68ee1b541973bb8e1144db23a194b
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/shirou/gopsutil to c95755e4bcd7a62bb8bd33f3a597a7c7f35e2cf3
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/shirou/w32 to 3c9377fc6748f222729a8270fe2775d149a249ad
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/Shopify/sarama to 3b1b38866a79f06deddf0487d5c27ba0697ccd65
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/Sirupsen/logrus to 61e43dc76f7ee59a82bdf3d71033dc12bea4c77d
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/soniah/gosnmp to f15472a4cd6f6ea7929e4c7d9f163c49f059924f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/StackExchange/wmi to f3e2bae1e0cb5aef83e319133eabfee30013a4a5
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/streadway/amqp to 63795daa9a446c920826655f26ba31c81c860fd6
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/stretchr/objx to facf9a85c22f48d2f52f2380e4efce1768749a89
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/stretchr/testify to 12b6f73e6084dad08a7c6e575284b177ecafbc71
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/tidwall/gjson to 0623bd8fbdbf97cc62b98d15108832851a658e59
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/tidwall/match to 173748da739a410c5b0b813b956f89ff94730b4c
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/vjeantet/grok to d73e972b60935c7fec0b4ffbc904ed39ecaf7efe
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/wvanbergen/kafka to bc265fedb9ff5b5c5d3c0fdcef4a819b3523d3ee
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/wvanbergen/kazoo-go to 968957352185472eacb69215fa3dbfcfdbac1096
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/yuin/gopher-lua to 66c871e454fcf10251c61bf8eff02d0978cae75a
> Restoring /home/omangold/tmp/telegraf/mybuild/src/github.com/zensqlmonitor/go-mssqldb to ffe5510c6fa5e15e6d983210ab501c815b56b363
> Restoring /home/omangold/tmp/telegraf/mybuild/src/golang.org/x/crypto to dc137beb6cce2043eb6b5f223ab8bf51c32459f4
> Restoring /home/omangold/tmp/telegraf/mybuild/src/golang.org/x/net to f2499483f923065a842d38eb4c7f1927e6fc6e6d
> Restoring /home/omangold/tmp/telegraf/mybuild/src/golang.org/x/sys to 739734461d1c916b6c72a63d7efda2b27edb369f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/golang.org/x/text to 506f9d5c962f284575e88337e7d9296d27e729d3
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/asn1-ber.v1 to 4e86f4367175e39f69d9358a5f17b4dda270378d
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/fatih/pool.v2 to 6e328e67893eb46323ad06f0e92cb9536babbabc
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/gorethink/gorethink.v3 to 7ab832f7b65573104a555d84a27992ae9ea1f659
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/ldap.v2 to 8168ee085ee43257585e50c6441aadf54ecb2c9f
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/mgo.v2 to 3f83fa5005286a7fe593b055f0d7771a7dce4655
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/olivere/elastic.v5 to 3113f9b9ad37509fe5f8a0e5e91c96fdc4435e26
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/tomb.v1 to dd632973f1e7218eb1089048e0798ec9ae7dceb8
> Restoring /home/omangold/tmp/telegraf/mybuild/src/gopkg.in/yaml.v2 to 4c78c975fe7c825c6d1466c42be594d1d6f3aba6

ERROR restoring some imports:
-   Error checking out rev ea9585611a4ab58a205b9b125ebd74c389a6b898 of repo /home/omangold/tmp/telegraf/mybuild/src/github.com/nats-io/nats, exit status 128
make[1]: *** [Makefile:29: deps] Error 1
make[1]: Leaving directory '/home/omangold/tmp/telegraf/mybuild/src/github.com/influxdata/telegraf'
make: *** [Makefile:23: all] Error 2
```

### Additional info:

Looks to me like nats-io/nats has been renamed to nats-io/go-nats and nats-io/nats only hosts docs now.

**Comment by nicgrobler** [comment]
just found this as well - as mentioned above. changed "nats" to "go-nats" in inputs and outputs fixes the build.

**Comment by danielnelson** [comment]
I added the fix to the release-1.6 branch, but there will probably not be another 1.6.x release.
